bentoml/_internal/bento/docker/init.sh

```diff
@@ -20,9 +20,9 @@ if [ $# -eq 0 ] || [ $1 == "ensure_python" ] ; then
   if [ -f ./env/python/version.txt ]; then
     PY_VERSION_SAVED=$(cat ./env/python/version.txt)
     # remove PATCH version - since most patch version only contains backwards compatible
-    # bug fixes and the BentoML defautl docker base image will include the latest
+    # bug fixes and the BentoML default docker base image will include the latest
     # patch version of each Python minor release
-    DESIRED_PY_VERSION=${PY_VERSION_SAVED:0:3} # returns 3.7, 3.8 or 3.9
+    DESIRED_PY_VERSION=${PY_VERSION_SAVED%.*} # returns major.minor
```

Comment:
This will return the `major.minor.patch` right?  e.g. if `PY_VERSION_SAVED=3.10.12`, it will return `3.10.12` here? is it possible to use `grep` or `sed` to extract only the `major.minor` here?  Since BentoML only pre-built base image for all the python minor versions supported, but not for all the `patch` versions.

Reply:
This will only return `major.minor`. But we eventually want `major.minor.patch`

Reply:
nvm, just realize what `%.*` does in bash